app/models/post_action.rb

```diff
@@ -239,8 +239,8 @@ def update_counters
     end
 
     if column == "like_count"
-      topic_count = Post.where(topic_id: topic_id).sum(column)
-      Topic.where(id: topic_id).update_all ["#{column} = ?", topic_count]
+      topic = Topic.find_by(id: topic_id)
+      topic.update_action_counts if topic
```

Comment:
How about this one-liner instead? 🤔 

```suggestion
      Topic.find_by(id: topic_id)&.update_action_counts
```

Reply:
Have made the changes :+1: